lib/src/widgets/information_message/ds_information_message.widget.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+import 'package:blip_ds/blip_ds.dart';
+
+/// Messages and warnings to be shown in agent and customer actions
+///
+/// Parameters [chatbotIdentity] and [agentIdentity] are sent to personalize the message.
+/// Depending on the message type defined by the [messageType] parameter, other parameters
+/// may or may not be mandatory.
+class DSInformationMessage extends StatelessWidget {
+  /// include the parameter [ticketNumber] to be displayed if the message type is [DSInformationMessageType.forwardedTicket]
+  final String? ticketNumber;
+
+  /// Enter [chatbotIdentity] to compose the message if it is of type [DSInformationMessageType.forwardedTicket]
+  final String? chatbotIdentity;
+
+  /// include the parameter [agentIdentity] to identify the attendant to compose the message if
+  /// it is of type [DSInformationMessageType.closedAttendant]
+  final String? agentIdentity;
+
+  /// Possible message types can be [DSInformationMessageType.forwardedTicket],
+  /// [DSInformationMessageType.closedAttendant] or [DSInformationMessageType.closedCustomer]
+  final DSInformationMessageType messageType;
+
+  final _message = <String>[
+    'Chatbot {chatbotIdentity} encaminhou a conversa para atendimento',
+    'Atendente {agentIdentity} encerrou o atendimento',
+    'Cliente encerrou o atendimento',
+  ];
+
+  DSInformationMessage({
+    super.key,
+    this.ticketNumber,
+    this.agentIdentity,
+    this.chatbotIdentity,
+    required this.messageType,
+  })  : assert((messageType == DSInformationMessageType.forwardedTicket)
+            ? (ticketNumber != null &&
+                chatbotIdentity != null &&
+                ticketNumber.trim().isNotEmpty &&
+                chatbotIdentity.trim().isNotEmpty)
+            : true),
+        assert((messageType == DSInformationMessageType.closedAttendant)
+            ? (agentIdentity != null && agentIdentity.trim().isNotEmpty)
+            : true);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        DSBodyText(
+          _prepareMessage(messageType),
+          textAlign: TextAlign.center,
+          overflow: TextOverflow.visible,
+          fontWeight: FontWeight.w400,
+        ),
+        if (messageType == DSInformationMessageType.forwardedTicket)
+          Padding(
+            padding: const EdgeInsets.only(top: 5.0),
+            child: DSBodyText(
+              'Ticket $ticketNumber',
```

Comment:
we can have a suffix after the ticket number..
vue version:
<h3>Ticket #{{`${document.sequentialId}${document.sequentialSuffix ? document.sequentialSuffix : ''}`}}</h3>

Reply:
This ticket formatting could stay in the application, as leaving the DS just showing the ticket would open the possibility of some formatting different from the ticket. Despite this, the change was made.